src/library/scala/util/hashing/MurmurHash3.scala

```diff
@@ -210,14 +210,14 @@ object MurmurHash3 extends MurmurHash3 {
   final val setSeed         = "Set".hashCode
 
   def arrayHash[@specialized T](a: Array[T]): Int  = arrayHash(a, arraySeed)
-  def bytesHash(data: Array[Byte]): Int            = bytesHash(data, arraySeed)
+  def bytesHash(data: Array[Byte]): Int            = arrayHash(data, arraySeed)
   def orderedHash(xs: TraversableOnce[Any]): Int   = orderedHash(xs, symmetricSeed)
   def productHash(x: Product): Int                 = productHash(x, productSeed)
   def stringHash(x: String): Int                   = stringHash(x, stringSeed)
   def unorderedHash(xs: TraversableOnce[Any]): Int = unorderedHash(xs, traversableSeed)
 
   private[scala] def wrappedArrayHash[@specialized T](a: Array[T]): Int  = arrayHash(a, seqSeed)
-  private[scala] def wrappedBytesHash(data: Array[Byte]): Int            = bytesHash(data, seqSeed)
+  private[scala] def wrappedBytesHash(data: Array[Byte]): Int            = arrayHash(data, seqSeed)
```

Comment:
We could also delete `def wrappedBytesHash` method entirely because it is only used once (in WrappedArray.ofByte), and it is private, that caller could call `wrappedArrayHash`

Reply:
We probably could do so, but I would prefer any further cleanups be done in 2.13. Not sure what the conventions around scala development are, but it would be better to minimize changes in hotfixes. (I had broken this rule myself previously, of course...)